### PR TITLE
postgresql added

### PR DESCRIPTION
**Pull request by tima7-code**

### CODE REVIEW

**Review by ZuxiGIT — DISMISSED** [review]
LGTM

File: docker-compose.yaml [hunk 1]
```diff
@@ -8,8 +8,21 @@ services:
...
     ports:
       - "8080:8000"
+  db:
+    image: postgres:13.0-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=goald
+      - POSTGRES_PASSWORD=goald
+      - POSTGRES_DB=goald_db
```
[the remark this answers is not in the document]
Comment (uf00led) [reply docker-compose.yaml]: bad creds, but ok for pet-project

**Review by uf00led — APPROVED** [review]
Good Job! Now PostgreSQL in our projects

**Review by ZuxiGIT — COMMENTED** [review]
LGTM

**Review by ZuxiGIT — APPROVED** [review]

**Pull request merged**